### ISSUE TITLE
keyframes selectorlists not unfolded

### ISSUE THREAD

**Issue by bartveneman** [projectwallace/css-analyzer#496]
blocks https://github.com/projectwallace/projectwallace.com/pull/109

```css
keyframes Toastify__bounceInRight {
	0%,
	60%,
	75%,
	90%,
	to {
		animation-timing-function: cubic-bezier(.215, .61, .355, 1);
	}
```

`0%, 60%, 75%, 90%, to` is reported as one single selector in `selectors.keyframes.unique`
